Package.swift

```diff
@@ -5,7 +5,7 @@ import PackageDescription
 
 let package = Package(
     name: "ObservableStore",
-    platforms: [.macOS(.v10_15), .iOS(.v15)],
+    platforms: [.macOS(.v11), .iOS(.v15)],
```

Comment:
Not sure if this is intended? I think macOS 11 is probably a safe target by now though.

Reply:
Necessary. Log message privacy only available on macOS on or after this version.